BraveWallet/Extensions/BraveWalletSwiftUIExtensions.swift

```diff
@@ -42,6 +42,30 @@ extension BraveWallet.NetworkInfo: Identifiable {
     )
   }
   
+  public var nativeTokenLogo: String {
+    if symbol.caseInsensitiveCompare("ETH") == .orderedSame {
+      return "eth-asset-icon"
+    } else if symbol.caseInsensitiveCompare("SOL") == .orderedSame {
+      return "sol-asset-icon"
+    } else if symbol.caseInsensitiveCompare("FIL") == .orderedSame {
+      return "filecoin-asset-icon"
+    } else if chainId.caseInsensitiveCompare(BraveWallet.PolygonMainnetChainId) == .orderedSame {
+      return "matic"
+    } else if chainId.caseInsensitiveCompare(BraveWallet.BinanceSmartChainMainnetChainId) == .orderedSame {
+      return "bnb-asset-icon"
+    } else if chainId.caseInsensitiveCompare(BraveWallet.CeloMainnetChainId) == .orderedSame {
+      return "celo"
+    } else if chainId.caseInsensitiveCompare(BraveWallet.AvalancheMainnetChainId) == .orderedSame {
+      return "avax"
+    } else if chainId.caseInsensitiveCompare(BraveWallet.FantomMainnetChainId) == .orderedSame {
+      return "fantom"
+    } else if chainId.caseInsensitiveCompare(BraveWallet.OptimismMainnetChainId) == .orderedSame {
+      return "optimism" // should never happen since OptimismMainnet's symbol is ETH
+    } else {
+      return iconUrls.first ?? ""
```

Comment:
Kind of seems like we should use an optional String for this property, otherwise may have to check against `isEmpty` / empty string later?

Reply:
fixed in [1bc5c0f](https://github.com/brave/brave-ios/pull/5772/commits/1bc5c0fb0fc17be231c07cdd3804706b9c141410)